checkstyle/suppressions.xml

```diff
@@ -361,4 +361,7 @@
     <suppress checks="(ClassDataAbstractionCoupling|ClassFanOutComplexity)"
               files="(ReplicaFetcherThreadBenchmark).java"/>
 
+    <!--import order -->
+    <suppress checks="ImportOrder" files="^*$"/>
```

Comment:
why we need this change?

Reply:
I think checkstyle should consistent with auto format . If you open the A module auto format, we should open the module check rule.
So i add this line in order that open rule some module in future

Reply:
Making the consistency is extra effort. Also, those changes will complicate checkstyle files. Hence, I prefer to remove them if we do NOT use it actually.